beeline/src/java/org/apache/hive/beeline/DatabaseConnection.java

```diff
@@ -59,10 +58,8 @@ public boolean isClosed() {
     return (null == connection);
   }
 
-  public DatabaseConnection(BeeLine beeLine, String driver, String url,
-       Properties info) throws SQLException {
+  public DatabaseConnection(BeeLine beeLine, String url, Properties info) throws SQLException {
```

Comment:
From the `DatabaseConnection` constructor code it won't throw any exception so `throws SQLException` can be removed.